code/addons/test/src/components/Interaction.tsx

```diff
@@ -19,6 +19,8 @@ const MethodCallWrapper = styled.div(() => ({
   fontSize: typography.size.s1,
   overflowWrap: 'break-word',
   inlineSize: 'calc( 100% - 40px )',
+  // TODO: discuss about this
```

Comment:
It's still a to-do? Do you want to discuss it before merging the PR?

Reply:
I'll remove and talk to Arestad about this